src/Sarif.Multitool/Rules/SarifValidationSkimmerBase.cs

```diff
@@ -186,7 +214,7 @@ internal static string JsonPointerToJavaScript(string pointerString)
             var pointer = new JsonPointer(pointerString);
             foreach (string token in pointer.ReferenceTokens)
             {
-                if (int.TryParse(token, out int index))
+                if (int.TryParse(token, out _))
```

Comment:
>_ [](start = 44, length = 1)

NIT can you use 'unusedIndex' or something rather than an underscore? i find it jarring to see this where I expect a variable name. I understand you're trying to indicate it is not consumed.

Reply:
Actually this is a language feature. "`_`" is special -- note how I didn't have to specify the type of the output parameter.

---
In reply to: [305430798](https://github.com/microsoft/sarif-sdk/pull/1570#discussion_r305430798) [](ancestors = 305430798)

Reply:
aah, beautiful!! TIL. I always found it annoying thing about TryParses! most of the times we dont care about the out param, so this makes me very happy! :)

---
In reply to: [305525416](https://github.com/microsoft/sarif-sdk/pull/1570#discussion_r305525416) [](ancestors = 305525416,305430798)